PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -87,4 +87,41 @@ private static void GenerateQuestionsFile(string filePath, int numberOfQuestions
             File.AppendAllLines(filePath, lines);
         }
     }
+
+    [TestMethod]
+    [DataRow("y",true)]
+    [DataRow("n", false)]
+    [DataRow("Z\nn", false)] //fail
+    [DataRow("adsf\nn", false)] //fail
+    [DataRow("[][]\ny", true)]
+    [DataRow("234)(\ny", true)]
+    public void RestartQuiz_UserInput_ReturnsExpected(String userInput,bool expected)
+    {
+        var reader = new StringReader(userInput);
+        //arrange
+        bool returnVar;
+        Console.SetIn(reader);
+        //act
+        returnVar = Program.RestartQuiz();
+        //assert
+        Assert.AreEqual(returnVar,expected,"Method returned: "+returnVar+ " expected: "+expected);
+    }
+
+    [TestMethod]
+    [DataRow("1\n1\n1\n1\n1\n1\n1\nn\n")]
+    public void Main_UserInput_ConsoleOutputContainsRetakePrompt(String userInput)
```

Comment:
https://essentialcsharp.com/guidelines

DO use the C# keyword rather than the unqualified name when specifying a data type (e.g., string rather than String).
```suggestion
    public void Main_UserInput_ConsoleOutputContainsRetakePrompt(string userInput)
```